README.md

```diff
@@ -39,7 +39,8 @@ $ pip install tap-asana
   "client_secret": "xxx",
   "redirect_uri": "urn:ietf:wg:oauth:2.0:oob",
```

Comment:
This PR will be merged after PR #41, as some code is common between the branches.